scicomp/glossary.rst

```diff
@@ -0,0 +1,51 @@
+Glossary of scientific computing
+================================
+
+.. glossary::
+   :sorted:
+
+   Automated testing
+       See testing
+
+   Command line
+
+   Computational complexity
+
+   Documentation
+
```

Comment:
Comments and other information provided by a programmer towards the use of code or other resources.